lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -306,29 +307,34 @@ export class FormFieldModel extends FormWidgetModel {
         /*
          This is needed due to Activiti issue related to reading dropdown values as value string
          but saving back as object: { id: <id>, name: <name> }
+         Side note: Probably not valid anymore
          */
         if (json.type === FormFieldTypes.DROPDOWN) {
-            if (json.options) {
-                if (json.hasEmptyValue) {
-                    const emptyOption = json.options[0];
-                    if (value === '' || value === emptyOption.id || value === emptyOption.name) {
-                        value = emptyOption.id;
-                    }
-                } else {
-                    if (value?.id && value?.name) {
-                        value = value.id;
-                    }
-                }
+            if (this.hasEmptyValue && value === null) {
+                value = this.emptyValueOption;
+                return value;
```

Comment:
why do u return assign and return value instead of just returning this.emptyOption?

Reply:
Thx, forgot to change it